src/components/Checkbox/__stories__/checkbox.stories.mdx

```diff
@@ -0,0 +1,168 @@
+<Canvas>
+  <Story name="Overview"
+         args={ {label: "Option", checked: true} }>
+    { checkboxTemplate.bind({}) }
+  </Story>
+</Canvas>
+
+## Props
+<ArgsTable of={ Checkbox } />
+
+## Usage
+<UsageGuidelines guidelines={[
+  <span class="monday-style-story-checkbox_list">Use checkboxes to:
+    <span class="monday-style-story-checkbox_list-item">1. Select one or more options from a list</span>
+    <span class="monday-style-story-checkbox_list-item">2. Turn an item on or off in a desktop environment</span>
+  </span>,
+  "Use checkboxes independently from each other: selecting one checkbox shouldn’t change the selection status of another checkbox in the list. The exception is when a checkbox is used to make a bulk selection.",
+  "Ensure both label and input are clickable to select the checkbox field. ",
+  "Keep a positive tone of voice. For example: “Turn on notifications” instead of “Turn off notifications”.",
+  "Checkboxes should be listed according to a logical order.",
+  "Place checkboxes vertically, using 16px spacing.",
+  "Checkbox will always apear with a label.",
+]}/>
+
+<Tip title="Am I using the right component?">For settings that are immediately applied, consider using a <Link href="/?path=/docs/components-toggle--overview" size={Link.sizes.SMALL} withoutSpacing>toggle switch</Link> instead.</Tip>
+
+## Variants
+### States
+Has 4 states: regular, hover, selected, and disabled.
+<Canvas>
+  <Story name="States">
+    <Checkbox label="Regular" />
+    <Checkbox label="Selected" checked />
+    <Checkbox label="Disabled" disabled />
+    <Checkbox label="Indeterminate" indeterminate />
+  </Story>
+</Canvas>
+
+## Do’s and Don’ts
+<ComponentRules
+  rules={[
+    {
+      positive: {
+        component:
+          <DialogContentContainer className="monday-style-story-checkbox_wrapper">
+            <Checkbox label="Item 1" />
+            <Checkbox label="Item 2" checked />
+            <Checkbox label="Item 3" checked />
+          </DialogContentContainer>,
+        description:"Use checkboxes when one or more items can be selected."
+      },
+      negative: {
+        component: 
+          <DialogContentContainer className="monday-style-story-checkbox_wrapper">
+            <Checkbox label="Item 1" checked />
+            <Checkbox label="Item 2" disabled />
+            <Checkbox label="Item 3" disabled />
+          </DialogContentContainer>,
+        description:"Don't use checkboxes when only one item can be selected from a list. Use radio buttons instead."
+      }
+    },
+    {
+      positive: {
+        component:<div className="monday-style-story-checkbox_border"><Checkbox label="Option" checked /></div>,
+        description:"Use the checkbox label’s prop to describe the option purpse."
+      },
+      negative: {
+        component: [
+          <div className="monday-style-story-checkbox_border">
+            <Checkbox componentClassName="monday-style-story-checkbox_minus-margin" checked />
```

Comment:
We add support for "className" prop instead of "componentClassName" for checkbox (for consist API with the other components)
could you change the usage in the "componentClassName" in this component to "className"? (because "componentClassName" is deprecated)